README.md

```diff
@@ -119,16 +128,12 @@ for Linux, Postgres.app for MacOS)
 ```bash
 git clone git@github.com:timescale/timescaledb.git
 cd timescaledb
-
-# Find the latest release and checkout, e.g. for 0.8.0:
-git checkout 0.8.0
-
+# Find the latest release and checkout, e.g. for 1.0.0:
+git checkout 1.0.0
```

Comment:
Is there a reason why you don't use 1.1.0 here?

Reply:
Thought it `1.0.0` seemed like a good default value instead of trying to keep it up to date with each new release. Can change